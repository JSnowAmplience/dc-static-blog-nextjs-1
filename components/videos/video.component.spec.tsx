import renderer from 'react-test-renderer';
import VideoComponent from './video.component';
import Video from '../../common/interfaces/video.interface';
import { MediaType } from '../../common/interfaces/media.interface';

describe('VideoComponent', (): void => {
  it('should render an video', (): void => {
    const video: Video = {
      video: {
        defaultHost: 'i1-qa.adis.ws',
        endpoint: 'bloblogltd',
        name: 'SampleVideo_1280x720_5mb',
        id: '721044de-d125-4a1a-8ddc-2201b9463f2d',
        mediaType: MediaType.VIDEO
      }
    };

    const wrapper = renderer.create(<VideoComponent video={video.video} />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
