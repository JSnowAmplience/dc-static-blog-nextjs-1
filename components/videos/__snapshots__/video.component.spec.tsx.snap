// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoComponent should render an video 1`] = `
<video
  controls={true}
>
  <source
    src="//i1-qa.adis.ws/v/bloblogltd/SampleVideo_1280x720_5mb"
  />
</video>
`;
